Procfile

```diff
@@ -1,13 +1,12 @@
-controller: [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "HOSTNAME=testappcontroller-1  FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-application-controller $COMMAND --loglevel debug --redis localhost:${ARGOCD_E2E_REDIS_PORT:-6379} --repo-server localhost:${ARGOCD_E2E_REPOSERVER_PORT:-8081} --otlp-address=${ARGOCD_OTLP_ADDRESS} --application-namespaces=${ARGOCD_APPLICATION_NAMESPACES:-''} --server-side-diff-enabled=${ARGOCD_APPLICATION_CONTROLLER_SERVER_SIDE_DIFF:-'false'}"
-api-server: [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-server $COMMAND --loglevel debug --redis localhost:${ARGOCD_E2E_REDIS_PORT:-6379} --disable-auth=${ARGOCD_E2E_DISABLE_AUTH:-'true'} --insecure --dex-server http://localhost:${ARGOCD_E2E_DEX_PORT:-5556} --repo-server localhost:${ARGOCD_E2E_REPOSERVER_PORT:-8081} --port ${ARGOCD_E2E_APISERVER_PORT:-8080} --otlp-address=${ARGOCD_OTLP_ADDRESS} --application-namespaces=${ARGOCD_APPLICATION_NAMESPACES:-''}"
+controller: [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "GOCOVERDIR=${ARGOCD_COVERAGE_DIR:-/tmp/coverage} HOSTNAME=testappcontroller-1  FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-application-controller $COMMAND --loglevel debug --redis localhost:${ARGOCD_E2E_REDIS_PORT:-6379} --repo-server localhost:${ARGOCD_E2E_REPOSERVER_PORT:-8081} --otlp-address=${ARGOCD_OTLP_ADDRESS} --application-namespaces=${ARGOCD_APPLICATION_NAMESPACES:-''} --server-side-diff-enabled=${ARGOCD_APPLICATION_CONTROLLER_SERVER_SIDE_DIFF:-'false'}"
```

Comment:
my understanding of the `Procfile` and the creative shell here will launch the `./dist/argocd` binary in the [makefile](https://github.com/argoproj/argo-cd/blob/7cf5ed06d4f8105812a9430f52419179762350da/Makefile#L466) because `BIN_MODE` is [true](https://github.com/argoproj/argo-cd/blob/7cf5ed06d4f8105812a9430f52419179762350da/Makefile#L64). I also did a `ps -ef` and confirmed I did see the binaries.

It probably wouldn't hurt? 

```
Run ps -ef | grep argo
runner     16701    4934  0 13:05 ?        00:00:00 /bin/sh -c ARGOCD_SSH_DATA_PATH=/tmp/argo-e2e/app/config/ssh \ ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls \ ARGOCD_GPG_DATA_PATH=/tmp/argo-e2e/app/config/gpg/source \ ARGOCD_GNUPGHOME=/tmp/argo-e2e/app/config/gpg/keys \ ARGOCD_GPG_ENABLED=true \ ARGOCD_PLUGINCONFIGFILEPATH=/tmp/argo-e2e/app/config/plugin \ ARGOCD_PLUGINSOCKFILEPATH=/tmp/argo-e2e/app/config/plugin \ ARGOCD_E2E_DISABLE_AUTH=false \ ARGOCD_ZJWT_FEATURE_FLAG=always \ ARGOCD_IN_CI=true \ BIN_MODE=true \ ARGOCD_APPLICATION_NAMESPACES=argocd-e2e-external,argocd-e2e-external-2 \ ARGOCD_APPLICATIONSET_CONTROLLER_NAMESPACES=argocd-e2e-external,argocd-e2e-external-2 \ ARGOCD_APPLICATIONSET_CONTROLLER_ALLOWED_SCM_PROVIDERS=[http://127.0.0.1:8341,http://127.0.0.1:8342,http://127.0.0.1:8343,http://127.0.0.1:8344](http://127.0.0.1:8341,http//127.0.0.1:8342,http://127.0.0.1:8343,http://127.0.0.1:8344) \ ARGOCD_E2E_TEST=true \ ?goreman -f Procfile start 
runner     16713   16702  0 13:05 ?        00:00:00 /bin/sh -c [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "GOCOVERDIR=${ARGOCD_COVERAGE_DIR:-/tmp/coverage} HOSTNAME=testappcontroller-1  FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-application-controller $COMMAND --loglevel debug --redis localhost:${ARGOCD_E2E_REDIS_PORT:-6379} --repo-server localhost:${ARGOCD_E2E_REPOSERVER_PORT:-8081} --otlp-address=${ARGOCD_OTLP_ADDRESS} --application-namespaces=${ARGOCD_APPLICATION_NAMESPACES:-''} --server-side-diff-enabled=${ARGOCD_APPLICATION_CONTROLLER_SERVER_SIDE_DIFF:-'false'}"
runner     16715   16702  0 13:05 ?        00:00:00 /bin/sh -c [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "GOCOVERDIR=${ARGOCD_COVERAGE_DIR:-/tmp/coverage} FORCE_LOG_COLORS=4 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_BINARY_NAME=argocd-notifications $COMMAND --loglevel debug --application-namespaces=${ARGOCD_APPLICATION_NAMESPACES:-''} --self-service-notification-enabled=${ARGOCD_NOTIFICATION_CONTROLLER_SELF_SERVICE_NOTIFICATION_ENABLED:-'false'}"
runner     167[21](https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070#step:16:22)   16715  0 13:05 ?        00:00:00 sh -c GOCOVERDIR=/tmp/coverage FORCE_LOG_COLORS=4 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls ARGOCD_BINARY_NAME=argocd-notifications ./dist/argocd --loglevel debug --application-namespaces=argocd-e2e-external,argocd-e2e-external-2 --self-service-notification-enabled='false'
runner     16722   16702  0 13:05 ?        00:00:00 /bin/sh -c [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "GOCOVERDIR=${ARGOCD_COVERAGE_DIR:-/tmp/coverage} FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-server $COMMAND --loglevel debug --redis localhost:${ARGOCD_E2E_REDIS_PORT:-6379} --disable-auth=${ARGOCD_E2E_DISABLE_AUTH:-'true'} --insecure --dex-server http://localhost:${ARGOCD_E2E_DEX_PORT:-5556} --repo-server localhost:${ARGOCD_E2E_REPOSERVER_PORT:-8081} --port ${ARGOCD_E2E_APISERVER_PORT:-8080} --otlp-address=${ARGOCD_OTLP_ADDRESS} --application-namespaces=${ARGOCD_APPLICATION_NAMESPACES:-''}"
runner     16723   16721  0 13:05 ?        00:00:00 ./dist/argocd --loglevel debug --application-namespaces=argocd-e2e-external,argocd-e2e-external-2 --self-service-notification-enabled=false
runner     16725   16702  0 13:05 ?        00:00:00 /bin/sh -c sh -c "ARGOCD_BINARY_NAME=argocd-dex go run github.com/argoproj/argo-cd/v2/cmd gendexcfg -o `pwd`/dist/dex.yaml && (test -f dist/dex.yaml || { echo 'Failed to generate dex configuration'; exit 1; }) && docker run --rm -p ${ARGOCD_E2E_DEX_PORT:-5556}:${ARGOCD_E2E_DEX_PORT:-5556} -v `pwd`/dist/dex.yaml:/dex.yaml ghcr.io/dexidp/dex:$(grep "image: ghcr.io/dexidp/dex" manifests/base/dex/argocd-dex-server-deployment.yaml | cut -d':' -f3) dex serve /dex.yaml"
runner     16728   167[22](https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070#step:16:23)  0 13:05 ?        00:00:00 sh -c GOCOVERDIR=/tmp/coverage FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls ARGOCD_SSH_DATA_PATH=/tmp/argo-e2e/app/config/ssh ARGOCD_BINARY_NAME=argocd-server ./dist/argocd --loglevel debug --redis localhost:6379 --disable-auth=false --insecure --dex-server http://localhost:5556 --repo-server localhost:8081 --port 8088 --otlp-address= --application-namespaces=argocd-e2e-external,argocd-e2e-external-2
runner     16734   16702  0 13:05 ?        00:00:00 /bin/sh -c [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "GOCOVERDIR=${ARGOCD_COVERAGE_DIR:-/tmp/coverage} FORCE_LOG_COLORS=4 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-applicationset-controller $COMMAND --loglevel debug --metrics-addr localhost:1[23](https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070#step:16:24)45 --probe-addr localhost:12346 --argocd-repo-server localhost:${ARGOCD_E2E_REPOSERVER_PORT:-8081}"
runner     16735   16702  0 13:05 ?        00:00:00 /bin/sh -c [ "$BIN_MODE" = 'true' ] && COMMAND=./dist/argocd || COMMAND='go run ./cmd/main.go' && sh -c "GOCOVERDIR=${ARGOCD_COVERAGE_DIR:-/tmp/coverage} FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_GNUPGHOME=${ARGOCD_GNUPGHOME:-/tmp/argocd-local/gpg/keys} ARGOCD_PLUGINSOCKFILEPATH=${ARGOCD_PLUGINSOCKFILEPATH:-./test/cmp}  ARGOCD_GPG_DATA_PATH=${ARGOCD_GPG_DATA_PATH:-/tmp/argocd-local/gpg/source} ARGOCD_TLS_DATA_PATH=${ARGOCD_TLS_DATA_PATH:-/tmp/argocd-local/tls} ARGOCD_SSH_DATA_PATH=${ARGOCD_SSH_DATA_PATH:-/tmp/argocd-local/ssh} ARGOCD_BINARY_NAME=argocd-repo-server ARGOCD_GPG_ENABLED=${ARGOCD_GPG_ENABLED:-false} $COMMAND --loglevel debug --port ${ARGOCD_E2E_REPOSERVER_PORT:-8081} --redis localhost:${ARGOCD_E2E_REDIS_PORT:-6379} --otlp-address=${ARGOCD_OTLP_ADDRESS}"
runner     16736   16713  0 13:05 ?        00:00:00 sh -c GOCOVERDIR=/tmp/coverage HOSTNAME=testappcontroller-1  FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls ARGOCD_SSH_DATA_PATH=/tmp/argo-e2e/app/config/ssh ARGOCD_BINARY_NAME=argocd-application-controller ./dist/argocd --loglevel debug --redis localhost:6379 --repo-server localhost:8081 --otlp-address= --application-namespaces=argocd-e2e-external,argocd-e2e-external-2 --server-side-diff-enabled='false'
runner     16737   16702  0 13:05 ?        00:00:00 /bin/sh -c bash -c "if [ \"$ARGOCD_REDIS_LOCAL\" = 'true' ]; then redis-server --save '' --appendonly no --port ${ARGOCD_E2E_REDIS_PORT:-6379}; else docker run --rm --name argocd-redis -i -p ${ARGOCD_E2E_REDIS_PORT:-6379}:${ARGOCD_E2E_REDIS_PORT:-6379} docker.io/library/redis:$(grep "image: redis" manifests/base/redis/argocd-redis-deployment.yaml | cut -d':' -f3) --save '' --appendonly no --port ${ARGOCD_E2E_REDIS_PORT:-6379}; fi"
runner     16738   16728  0 13:05 ?        00:00:00 ./dist/argocd --loglevel debug --redis localhost:6379 --disable-auth=false --insecure --dex-server http://localhost:5556 --repo-server localhost:8081 --port 8088 --otlp-address= --application-namespaces=argocd-e2e-external,argocd-e2e-external-2
runner     16741   16736  0 13:05 ?        00:00:00 ./dist/argocd --loglevel debug --redis localhost:6379 --repo-server localhost:8081 --otlp-address= --application-namespaces=argocd-e2e-external,argocd-e2e-external-2 --server-side-diff-enabled=false
runner     16760   16737  0 13:05 ?        00:00:00 bash -c if [ "" = 'true' ]; then redis-server --save '' --appendonly no --port 6379; else docker run --rm --name argocd-redis -i -p 6379:6379 docker.io/library/redis:7.0.15-alpine --save '' --appendonly no --port 6379; fi
+ make test-e2e-local
runner     16763   16735  0 13:05 ?        00:00:00 sh -c GOCOVERDIR=/tmp/coverage FORCE_LOG_COLORS=1 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_GNUPGHOME=/tmp/argo-e2e/app/config/gpg/keys ARGOCD_PLUGINSOCKFILEPATH=/tmp/argo-e2e/app/config/plugin  ARGOCD_GPG_DATA_PATH=/tmp/argo-e2e/app/config/gpg/source ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls ARGOCD_SSH_DATA_PATH=/tmp/argo-e2e/app/config/ssh ARGOCD_BINARY_NAME=argocd-repo-server ARGOCD_GPG_ENABLED=true ./dist/argocd --loglevel debug --port 8081 --redis localhost:6379 --otlp-address=
runner     16769   16763  2 13:05 ?        00:00:00 ./dist/argocd --loglevel debug --port 8081 --redis localhost:6379 --otlp-address=
runner     16770   16730  0 13:05 ?        00:00:00 docker run --name e2e-git --rm -i -p 2222:2222 -p 9080:9080 -p 9443:9443 -p 9444:9444 -p 9445:9445 -w /go/src/github.com/argoproj/argo-cd -v /home/runner/work/argo-cd/argo-cd:/go/src/github.com/argoproj/argo-cd -v /tmp:/tmp docker.io/argoproj/argo-cd-ci-builder:v1.0.0 bash -c goreman -f ./test/fixture/testrepos/Procfile start
runner     16782   167[25](https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070#step:16:26)  0 13:05 ?        00:00:00 sh -c ARGOCD_BINARY_NAME=argocd-dex go run github.com/argoproj/argo-cd/v2/cmd gendexcfg -o /home/runner/work/argo-cd/argo-cd/dist/dex.yaml && (test -f dist/dex.yaml || { echo 'Failed to generate dex configuration'; exit 1; }) && docker run --rm -p 5556:5556 -v /home/runner/work/argo-cd/argo-cd/dist/dex.yaml:/dex.yaml ghcr.io/dexidp/dex:v2.38.0 dex serve /dex.yaml
runner     16783   16734  0 13:05 ?        00:00:00 sh -c GOCOVERDIR=/tmp/coverage FORCE_LOG_COLORS=4 ARGOCD_FAKE_IN_CLUSTER=true ARGOCD_TLS_DATA_PATH=/tmp/argo-e2e/app/config/tls ARGOCD_SSH_DATA_PATH=/tmp/argo-e2e/app/config/ssh ARGOCD_BINARY_NAME=argocd-applicationset-controller ./dist/argocd --loglevel debug --metrics-addr localhost:12345 --probe-addr localhost:12346 --argocd-repo-server localhost:8081
runner     16789   16782 15 13:05 ?        00:00:02 go run github.com/argoproj/argo-cd/v2/cmd gendexcfg -o /home/runner/work/argo-cd/argo-cd/dist/dex.yaml
runner     16790   16783  0 13:05 ?        00:00:00 ./dist/argocd --loglevel debug --metrics-addr localhost:12345 --probe-addr localhost:12346 --argocd-repo-server localhost:8081
runner     16791   16760  0 13:05 ?        00:00:00 docker run --rm --name argocd-redis -i -p 6379:6379 docker.io/library/redis:7.0.15-alpine --save  --appendonly no --port 6379
runner     17045       1  2 13:05 ?        00:00:00 gpg-agent --homedir /tmp/argo-e2e/app/config/gpg/keys --use-standard-socket --daemon
root       17319   17315  0 13:05 ?        00:00:00 nginx: master process nginx -prefix=/go/src/github.com/argoproj/argo-cd -g daemon off; -c /go/src/github.com/argoproj/argo-cd/test/fixture/testrepos/nginx.conf
runner     17676   16950 10 13:05 ?        00:00:01 /home/runner/work/argo-cd/argo-cd/ui/node_modules/esbuild-linux-64/bin/esbuild --service=0.14.[27](https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070#step:16:28) --ping
runner     17896   16789 73 13:05 ?        00:00:05 /opt/hostedtoolcache/go/1.22.3/x64/pkg/tool/linux_amd64/compile -o /tmp/go-build4054411612/b514/_pkg_.a -trimpath /tmp/go-build4054411612/b514=> -p k8s.io/api/core/v1 -lang=go1.21 -complete -buildid wD9WqjtSzshQOFa92_dB/wD9WqjtSzshQOFa92_dB -goversion go1.22.3 -c=4 -nolocalimports -importcfg /tmp/go-build4054411612/b514/importcfg -pack /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/annotation_key_constants.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/doc.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/generated.pb.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/lifecycle.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/objectreference.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/register.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/resource.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/taint.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/toleration.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/types.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/types_swagger_doc_generated.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/well_known_labels.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/well_known_taints.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/api/core/v1/zz_generated.deepcopy.go
lxd        17933   16906  0 13:05 ?        00:00:00 /usr/bin/tini -- /usr/local/bin/argocd-notifications
lxd        17944   17933  1 13:05 ?        00:00:00 /usr/local/bin/argocd-notifications
runner     18150   16789 60 13:06 ?        00:00:00 /opt/hostedtoolcache/go/1.22.3/x64/pkg/tool/linux_amd64/compile -o /tmp/go-build4054411612/b531/_pkg_.a -trimpath /tmp/go-build4054411612/b531=> -p k8s.io/kube-openapi/pkg/validation/spec -lang=go1.19 -complete -buildid aJFTwsdVEg82Bq-gxLLK/aJFTwsdVEg82Bq-gxLLK -goversion go1.22.3 -c=4 -nolocalimports -importcfg /tmp/go-build4054411612/b531/importcfg -pack /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/contact_info.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/external_docs.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/gnostic.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/header.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/info.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/items.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/license.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/operation.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/parameter.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/path_item.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/paths.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/ref.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/response.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/responses.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/schema.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/security_scheme.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/swagger.go /home/runner/work/argo-cd/argo-cd/vendor/k8s.io/kube-openapi/pkg/validation/spec/tag.go
runner     18156   16789 23 13:06 ?        00:00:00 /opt/hostedtoolcache/go/1.22.3/x64/pkg/tool/linux_amd64/compile -o /tmp/go-build4054411612/b585/_pkg_.a -trimpath /tmp/go-build4054411612/b585=> -p github.com/prometheus/procfs -lang=go1.19 -complete -buildid z[32](https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070#step:16:33)-szYPyGa8xUQ0gt0o/z32-szYPyGa8xUQ0gt0o -goversion go1.22.3 -c=4 -nolocalimports -importcfg /tmp/go-build4054411612/b585/importcfg -pack /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/arp.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/buddyinfo.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/cmdline.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/cpuinfo.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/cpuinfo_x86.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/crypto.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/doc.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/fs.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/fs_statfs_type.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/fscache.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/ipvs.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/kernel_random.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/loadavg.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/mdstat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/meminfo.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/mountinfo.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/mountstats.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_conntrackstat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_dev.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_ip_socket.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_protocols.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_route.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_sockstat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_softnet.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_tcp.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_udp.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_unix.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_wireless.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/net_xfrm.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/netstat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_cgroup.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_cgroups.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_environ.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_fdinfo.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_interrupts.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_io.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_limits.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_maps.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_netstat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_ns.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_psi.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_smaps.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_snmp.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_snmp6.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_stat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_status.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/proc_sys.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/schedstat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/slab.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/softirqs.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/stat.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/swaps.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/thread.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/vm.go /home/runner/work/argo-cd/argo-cd/vendor/github.com/prometheus/procfs/zoneinfo.go
runner     18171   18169  0 13:06 ?        00:00:00 grep argo
```
https://github.com/argoproj/argo-cd/actions/runs/9465596433/job/26076803070

Reply:
Ah sure that does make sense